app/models/BannerDesign.scala

```diff
@@ -27,6 +27,13 @@ object GuardianRoundelDesign {
   implicit val decoder = deriveEnumerationDecoder[GuardianRoundelDesign]
 }
 
+case class HeaderImage(
```

Comment:
did you consider reusing `BannerDesignVisual.Image`?

Reply:
I think I may have dissuaded Rik from that because that type has a `kind` which isn't used/needed here. What do you think?

Reply:
Ok fair enough